nominees/project-konko.json

```diff
@@ -11,25 +14,23 @@
   "SDGs": [
     {
       "SDGNumber": 4,
-      "evidenceURL": "https://www.unicef.org/innovation/FundGraduate/NubianVR \n https://medium.com/making-nubianvr/nubianvrs-educational-vision-the-vision-the-road-map-7fd59ec8d486"
```

Comment:
I would preserve all the information on this field

Reply:
Resolved in commit aefad21

Reply:
Thank you!